src/lib/evil/unposix/sys/types.h

```diff
@@ -1,21 +1,25 @@
 #ifndef UNPOSIX_SYS_TYPES_H
 #define UNPOSIX_SYS_TYPES_H
 
+#include_next <sys/types.h>
+
 #ifdef _MSC_VER
 #ifndef WIN32_LEAN_AND_MEAN
 # define WIN32_LEAN_AND_MEAN
 #endif
 
 #include <BaseTsd.h>
+#include <crtdefs.h>
+
```

Comment:
As it is still a PR I think we can already work on #33 and add a `#undef WIN32_LEAN_AND_MEAN`